applications/StructuralMechanicsApplication/tests/adjoint_sensitivity_analysis_tests/adjoint_beam_structure_3d2n/beam_test_local_stress_adjoint_parameters.json

```diff
@@ -111,6 +111,18 @@
             }
         }
     }],
+    "element_sensitivity_domain_integration" : [{
```

Comment:
is there a reason you don't add this to `list_other_processes` and create a new list?

Reply:
no. I will move it to the `list_other_processes`.